lib/metadata/linux/MiqRpmPackages/Sqlite.rb

```diff
@@ -0,0 +1,68 @@
+require "metadata/linux/LinuxPackages"
+require "tmpdir"
+
+class MiqRpmPackages
+  class Sqlite < MiqRpmPackages
+    def initialize(fs, dbFile)
+      @fs           = fs
+      @db_file_path = dbFile
+
+      @rpmdb_tempdir = nil
+      @rpmdb_path    = nil
+
+      if fs.present?
```

Comment:
I had to change the specs to stub this, but I think .present? from active_support is not available at this context, so we either need to require it, or implement differently.

Reply:
I can replace with just `if fs`, other callers in smartstate just do `if @fs.nil?; else`